content/cloud/labels.md

```diff
@@ -0,0 +1,65 @@
+title = "Labels"
+template = "cloud_main"
+date = "2023-10-30T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/labels.md"
+
+---
+- [Labels Overview](#labels-overview)
+- [Benefits of labels](#benefits-of-labels)
+- [How To Create A Label](#how-to-create-a-label)
+- [How to Delete A Label](#how-to-delete-a-label)
+- [Next Steps](#next-steps) 
+
+# Labels Overview
+
+Spin applications are inherently ephermal resources; therefore, you don't necessarily want other Fermyon Cloud resources (such as NoOps SQL Databases) to share the exact same lifecycle. Furthermore, as you build and prototype your Spin application you may want to connect to other resources without necessarily having to recompile your application, but you still need some construct to explain the relationship between your Spin application and the other resource you need it to connect with. 
```

Comment:
> Spin applications are inherently ephermal resources;

Nit: ephermal => "ephemeral"

It could get confusing to refer to apps as resources when we're also using "resources" to refer to things apps can connect to. Perhaps you can just say:

> Spin applications are inherently ephemeral;


Perhaps I'm different from others here, but I would have never assumed resources would have their lifecycles tied to applications so the first point would kind of confuse me. The points that seem most relevant to me are:
* Apps need a way to refer to resources
* Different apps might refer to the logical same resources by different names
* Labels are way to indicate how an app refers to a resource

This relationship is dynamic and can change while your app is running. If you want your app to talk to a different database, change the link to refer to a different database.

Reply:
This is good feedback - I'm approaching this assuming the reader is familiar with the opinionated label experience with KV and default. However, if you approach the label concept with fresh eyes then the bullet points you shared resonate better.